manifests/fpm/config.pp

```diff
@@ -111,12 +111,6 @@
     mode    => '0644',
   }
 
-  ensure_resource('file', ['/var/run/php-fpm/', '/var/log/php-fpm/'], {
```

Comment:
The issue reports a problem with `/var/run/php-fpm/` not `/var/log/php-fpm/`.

Reply:
Well we could of course keep the `/var/log/php-fpm/` file resource. But if we let manage the run directory by the package management why shouldn't we rely on the package management to create the log directory too?
It's also possible to configure the pool to not use the directory (and it is hardcoded here and not dependent on other configurations).

Reply:
The current code allows for specify a different user/group from what the package management uses. By removing the log file directory you remove functionality from the module. 

Also the log file directory is not related to the issue with the pid file. By splitting up different functionality into different PR's, it makes it easier to merge.